components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/bean/roles/provider/Roles.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.dashboards.core.bean.roles.provider;
+
+import org.wso2.carbon.analytics.permissions.bean.Role;
+import org.wso2.carbon.config.annotation.Element;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * This is the bean class for Role in deployment yaml.
+ */
+public class Roles {
+
+    @Element(description = "list of dashboard creator roles")
+    private List<Role> creator = Collections.emptyList();
+
+    public List<Role> getCreator() {
+        return creator;
+    }
+
+    public void setCreator(List<Role> creator) {
```

Comment:
Is this setter going to be used? if not, lets remove it (to make `Roles` class immutable)